pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -1303,6 +1303,20 @@ func parseDeviceIoListConfig(getconfigCtx *getconfigContext,
 					"key: %s, value: %s", key, value)
 			}
 		}
+
+		if ioDevicePtr.Vflist != nil && ioDevicePtr.Vflist.VfCount > 0 {
+			port.Vfs.Count = uint8(ioDevicePtr.Vflist.VfCount)
+			port.Vfs.Data = make([]sriov.EthVF, ioDevicePtr.Vflist.VfCount)
+			for i, vf := range ioDevicePtr.Vflist.Data {
+				port.Vfs.Data[i] = sriov.EthVF{
+					Index:   uint8(vf.Index),
+					Mac:     vf.Mac,
+					VlanID:  uint16(vf.VlanId),
+					PciLong: port.Phyaddr.PciLong,
```

Comment:
I believe that each VF has a distinct PCI address, not shared with its PF.
In EVE we discover those PCI addresses [here](https://github.com/lf-edge/eve/blob/9.4/pkg/pillar/sriov/sriov.go#L61-L108).
So as I understand it, PciLong for VF is not supposed to be set here in parseconfig.go

Reply:
Ah, let unset this then.